vpr/src/base/vpr_context.h

```diff
@@ -269,12 +269,13 @@ struct DeviceContext : public Context {
      ********************************************************************/
     t_clock_arch* clock_arch;
 
-    /**
-     * @brief Name of rrgraph file read (if any).
-     *
-     * Used to determine when reading rrgraph if file is already loaded.
-     */
-    std::string read_rr_graph_filename;
+    /// @brief Name of rrgraph file read (if any).
+    /// Used to determine if the file is already loaded when reading rrgraph.
```

Comment:
"if the specified rr-graph file is already loaded, so we can avoid redundant reading of the rr-graph"